spec/views/student/index.html.erb_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+require 'spec_helper'
```

Comment:
rails_helper requires spec_helper. so you don't need to include spec_helper in any of these files.